targets/csharp/source/PlayFabSDK/UnittestRunner/UUnitTestRunner.cs

```diff
@@ -18,15 +18,19 @@ public class CsSaveRequest
 
         public static int Main(string[] args)
         {
+            int result = 1;
+            bool taskFinished = false;
             try
             {
-                MainTask(args).Wait(60000);
+                var mainTask = MainTask(args);
+                taskFinished = mainTask.Wait(120000);
```

Comment:
I'm assuming this is a timeout? Can we make this a named variable to make sure we know time units (I'm assuming this is milliseconds?)

Reply:
Done.